ansible/roles/dev/tasks/main.yml

```diff
@@ -11,35 +10,24 @@
       line: "StrictModes no"
   notify: restart sshd
 
-- name: Install Pulp dnf repository
+- name: Install Pulp package repository
   when: ansible_distribution == 'Fedora' and ansible_distribution_version|int < 24
   get_url:
       url: https://repos.fedorapeople.org/repos/pulp/pulp/fedora-pulp.repo
       dest: /etc/yum.repos.d/fedora-pulp.repo
 
 - name: Install dnf config-manager
-  dnf: name=dnf-command(config-manager) state=present
+  package: name=dnf-command(config-manager) state=present
 
 - name: Enable Pulp Nightly repository
-  command: dnf config-manager --set-enabled pulp-nightlies
+  command: package config-manager --set-enabled pulp-nightlies
   when: ansible_distribution == 'Fedora' and ansible_distribution_version|int < 24 and pulp_nightly_repo_enabled == false
 
-# These can go away when https://fedorahosted.org/spin-kickstarts/ticket/59 is fixed
-- stat:
-      path: /etc/sudoers.d/vagrant-nopasswd
-  name: Detect vagrant sudoers file
-  register: vagrant_nopasswd
-- lineinfile:
-      dest: "{{ vagrant_nopasswd.stat.path }}"
-      regexp: '^vagrant'
-      line: 'vagrant ALL=(ALL) NOPASSWD: ALL'
-      validate: 'visudo -cf %s'
-      mode: 0440
-  when: vagrant_nopasswd.stat.exists
-  name: Rejigger vagrant sudoers file
-
+  # TODO: It's not clear how this list was created, it would be great
+  #       to know why we install all of these, and (if possible) get them
+  #       captured as requirements in the project itself.
```

Comment:
In looking at this, a lot of these appear to be either packages needed to install python packages, python packages themselves, or developer-related things. I think I can probably sift through this, clean it up, and get a little more detail into the task name/comments so that this list isn't such a mystery.